bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -48,7 +53,7 @@ def execute(self):
         self.__install_build_dependencies()
         for config in self.test_config.integ_test["test-configs"]:
             security = self.__is_security_enabled(config)
-            status = self.__setup_cluster_and_execute_test_config(security)
+            status = self.__setup_cluster_and_execute_test_config(security, config)
```

Comment:
`security` is derived from `config`, don't need to pass both.

Reply:
Exactly so I tried moving the whole thing in local_cluster but looks like `__is_security_enabled` is being used in this class as well. What do you recommend should we do here?

Reply:
Instead of extracting `security` [here](https://github.com/opensearch-project/opensearch-build/blob/0a209a5e2deed01f49f2c4afcf563b1ef096a589/bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py#L50-L51), you could extract it inside `__setup_cluster_and_execute_test_config` and only pass config to the method